Mars Onboarding/Features/LanguageAndSkill.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Languages and Skills
+
+As a user I would like to add languages and skills to my profile so that 
+I can showcase my abilities to potential employers.
+@tag1
+Scenario: Add a new language to the profile
```

Comment:
please separate this into individual features ( skill feature and language feature)